_docs/concepts/traffic-management/rules-configuration.md

```diff
@@ -148,6 +148,96 @@ route:
   weight: 75
 ```
 
+### Timeouts and Retries
+
+By default, the timeout for http requests is 15 seconds,
+but this can be overridden in a route rule as follows:
+
+```yaml
+destination: "ratings.default.svc.cluster.local"
+route:
+- tags:
+    version: v1
+httpReqTimeout:
+  simpleTimeout:
+    timeoutSeconds: 10
+```
+
+The number of retries for a given http request can also be specified in a route rule.
+The maximum number of attempts, or as many as possible within the default or overriden timeout period,
+can be set as follows:
+
+```yaml
+destination: "ratings.default.svc.cluster.local"
+route:
+- tags:
+    version: v1
+httpReqRetries:
+  simpleRetry:
+    attempts: 3
+```
+
```

Comment:
Also mention:
The maximum number of retries can also be controlled on a per-request basis by providing the x-envoy-max-retries. A retry policy should be in place in order for this header field to be honored. 
In addition, setting x-envoy-upstream-rq-per-try-timeout-ms header (value in milliseconds) on outbound will cause Envoy to set a per try timeout. This allows a caller to set a tight per try timeout to allow for retries while maintaining a reasonable overall timeout. The per-try timeout specified via HTTP header must be <= the global route timeout or it will be ignored.